src/Core/src/Handlers/MenuFlyoutSubItem/MenuFlyoutSubItemHandler.cs

```diff
@@ -21,7 +21,8 @@ public partial class MenuFlyoutSubItemHandler : ElementHandler<IMenuFlyoutSubIte
 		{
 #if WINDOWS
 			[nameof(IMenuFlyoutSubItem.Text)] = MapText,
-			[nameof(IMenuFlyoutSubItem.Source)] = MapSource
+			[nameof(IMenuFlyoutSubItem.Source)] = MapSource,
+			[nameof(IMenuFlyoutSubItem.IsEnabled)] = MapIsEnabled,
```

Comment:
Found and fixed this nice little bug too! Will also fix in a separate PR just in case this one takes a bit longer to merge.